src/modules/components/transaction/transactionDataListItem/transactionDataListItemSkeleton/transactionDataListItemSkeleton.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { TransactionDataListItem } from '../../../..';
+import { DataList } from '../../../../../core';
+
+const meta: Meta<typeof TransactionDataListItem.Skeleton> = {
+    title: 'Modules/Components/Transaction/TransactionDataListItem.Skeleton',
```

Comment:
Could you update the `TransactionDataListItem.Structure` and `MemberDataListItem` stories as well? 🙏🏻 
```suggestion
    title: 'Modules/Components/Transaction/TransactionDataListItem/TransactionDataListItem.Skeleton',
```